src/Sylius/Bundle/WebBundle/Behat/WebContext.php

```diff
@@ -46,7 +46,7 @@ public function iShouldBeOnTheLoginPage()
     }
 
     /**
-     * @Given /^go to "([^""]*)" tab$/
+     * @Given /^(?:|I )go to "([^""]*)" tab$/
```

Comment:
So isn't ``@Given /^(?:|I )go to "([^""]*)" tab$/`` the best option? :smile: It can be used then to ``Given I go to ... tab``, ``When I go to ... tab`` as well as ``And go to ... tab`` etc.